mmv1/templates/terraform/examples/network_management_connectivity_test_cloudsql.tf.erb

```diff
@@ -0,0 +1,53 @@
+# [START networkmanagement_test_cloudsql]
+resource "google_network_management_connectivity_test" "<%= ctx[:primary_resource_id] %>" {
+  name = "<%= ctx[:vars]['primary_resource_name'] %>"
+  source {
+    cloud_sql_instance = google_sql_database_instance.source.self_link
+  }
+
+  destination {
+    cloud_sql_instance = google_sql_database_instance.dest.self_link
+  }
+
+  protocol = "TCP"
+  labels = {
+    env = "test"
+  }
+}
+
+resource "google_compute_network" "vpc" {
+  name = "<%= ctx[:vars]['network'] %>"
+}
+
+resource "google_sql_database_instance" "source" {
```

Comment:
```
    vcr_utils.go:152: Step 1/2 error: Error running apply: exit status 1
        
        Error: Error, failed to create instance because the network doesn't have at least 1 private services connection. Please see https://cloud.google.com/sql/docs/mysql/private-ip#network_requirements for how to create this connection.
        
          with google_sql_database_instance.source,
          on terraform_plugin_test.tf line 22, in resource "google_sql_database_instance" "source":
          22: resource "google_sql_database_instance" "source" {
        
        
        Error: Error, failed to create instance because the network doesn't have at least 1 private services connection. Please see https://cloud.google.com/sql/docs/mysql/private-ip#network_requirements for how to create this connection.
        
          with google_sql_database_instance.dest,
          on terraform_plugin_test.tf line 38, in resource "google_sql_database_instance" "dest":
          38: resource "google_sql_database_instance" "dest" {
```